requirements/dev.txt

```diff
@@ -261,10 +261,10 @@ cssutils==1.0.2 \
     --hash=sha256:a2fcf06467553038e98fea9cfe36af2bf14063eb147a70958cfcaa8f5786acaf \
     --hash=sha256:c74dbe19c92f5052774eadb15136263548dd013250f1ed1027988e7fef125c8d \
     # via premailer
-dataclasses==0.7 ; python_version < "3.7" \
-    --hash=sha256:3459118f7ede7c8bea0fe795bff7c6c2ce287d01dd226202f7c9ebc0610a7836 \
-    --hash=sha256:494a6dcae3b8bcf80848eea2ef64c0cc5cd307ffc263e17cdf42f3e5420808e6 \
-    # via -r requirements/common.in, libcst, pyre-check
+dataclasses==0.6 \
+    --hash=sha256:454a69d788c7fda44efd71e259be79577822f5e3f53f029a22d08004e951dc9f \
+    --hash=sha256:6988bd2b895eef432d562370bb707d540f32f7360ab13da45340101bc2307d84 \
+    # via pyre-check
```

Comment:
Downgrading `dataclasses` isn’t what we want. (Is this a result of running `update-locked-requirements` on Ubuntu 20.04? For compatibility, we only run that on 18.04.) I think you want to undo all the `requirements` changes; they aren’t needed.